core/src/main/java/io/grpc/inprocess/InProcessTransport.java

```diff
@@ -205,6 +206,11 @@ private synchronized void setListener(ClientStreamListener listener) {
       }
 
       @Override
+      public void setDefaultCodec(Codec c) {
+        // nop
```

Comment:
This was actually intentional.  For the inprocess transport, there is no Framer or deframer, so it would be a little wonky to set a codec.  If you think there might be a use for enabling coding on the in process I would be willing to add something in.